phoenix-core/src/it/java/org/apache/phoenix/end2end/index/IndexMetadataIT.java

```diff
@@ -610,7 +613,84 @@ public void testAsyncRebuildTimestamp() throws Exception {
                 "order by table_name" );
         assertFalse(rs.next());
     }
-    
+
+    @Test
+    public void testAsyncRebuildAll() throws Exception {
+        Properties props = PropertiesUtil.deepCopy(TEST_PROPERTIES);
+        Connection conn = DriverManager.getConnection(getUrl(), props);
```

Comment:
I would suggest using try-with-resources for connection and prepareStatement objects.